context/apply_action.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2024 Canonical Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package context
+
+type ApplyAction struct {
+	Dupl bool
+	Nocp bool
+	Buff bool
+	Forw bool
+	Drop bool
```

Comment:
Is there any reason for only having these fields? go-pfcp also supports `IPMA`, `IPMD`, `DFRT`, `EDRT`, `BDPN`, `DDPN`, `FSSM`, `MBSU`

Reply:
These are the same ones that were used in the pfcp library. I haven't added or removed any.

There are a bunch of ways we can still improve the SMF but here I only wanted to get rid of the PFCP library for IE's and PFCP messages. 

Reference:
- https://github.com/omec-project/pfcp/blob/master/pfcpType/ApplyAction.go#L12